internal/devserver/ts_devserver.bzl

```diff
@@ -194,13 +202,27 @@ def ts_devserver_macro(tags = [], **kwargs):
       **kwargs: passed through to `ts_devserver`
     """
     ts_devserver(
+        name = "%s_bin" % name,
+        data = data + ["@bazel_tools//tools/bash/runfiles"],
+        testonly = testonly,
+        visibility = ["//visibility:private"],
         # Users don't need to know that these tags are required to run under ibazel
         tags = tags + [
             # Tell ibazel not to restart the devserver when its deps change.
             "ibazel_notify_changes",
             # Tell ibazel to serve the live reload script, since we expect a browser will connect to
             # this program.
             "ibazel_live_reload",
-        ],
-        **kwargs
+        ],      
+      **kwargs
+    )
+
+    native.sh_binary(
```

Comment:
Windows itself does not run `.sh` files, so `native.sh_binary` is needed over a `.sh` file to converted it into a native binary.

Sometimes just using the `.sh` will work. But when it does work, it's because at some point that `.sh` file is run inside a MSYS or Cygwin environment by virtue of your machine setup. 

In the worst case scenario, the shell that ended up running is not the same as BAZEL_SH and you get errors like this: https://github.com/bazelbuild/bazel/issues/5724#issuecomment-410083088

Reply:
Good point. I remember seeing such an error on other machines w/ Bazel. 

I wonder if we still should make the `output.script` executable then. Since the rule implementation _basically_ just creates a Bash script (_source file_) that will be run by `sh_binary` then.

Reply:
That's a good point really. I agree, it shouldn't be an executable.

Reply:
Changed it to not be an executable.